pages/1.10/administering-clusters/deploying-a-local-dcos-universe/index.md

```diff
@@ -200,6 +200,9 @@ You can install and run DC/OS services on a datacenter without internet access w
     dcos package repo add local-universe http://master.mesos:8082/repo
     ```
 
+    **NOTE:** If you need help resolving `master.mesos`, refer to [Installing your own set of Universe packages](https://docs.mesosphere.com/1.9/administering-clusters/deploying-a-local-dcos-universe/#installing-your-own-set-of-universe-packages).
+
```

Comment:
Does not really link to FAQs page. However I went ahead and tried to find the approriate link but I could not find any details around resolving master.mesos on the FAQs page. I will keep looking and update here if I find it